Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml

```
@@ -67,6 +67,14 @@
   components:
   - type: Sprite
     state: poster2_contraband
+  - type: ExtendDescription
+    descriptionList:
+      - description: "mindshield-extenddescription-poster-atmosia"
+        fontSize: 12
+        color: "#11aaff"
+        requireDetailRange: false
+        requirements:
+        - !type:CharacterMindshieldRequirement
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 8

(indentation)